webapp/pages/settings/index.vue

```diff
@@ -88,8 +88,8 @@ export default {
     return {
       axiosSource: null,
       cities: [],
-      sending: false,
-      loading: false,
+      loadingData: false,
+      loadingGeo: false,
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-02T18:00:02Z" title="Thursday, May 2nd 2019, 8:00:02 pm +02:00">May 2, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -88,8 +88,8 @@ export default {
     return {
       axiosSource: null,
       cities: [],
-      sending: false,
-      loading: false,
+      loadingData: false,
+      loadingGeo: false,
```


This is such a common pattern in our application that I ask myself how we could DRY this up? Maybe with a `loading` mixin that gives us a method wrapper and handles the update flags. :thinking: